examples/shared_samples/connection_template.rb

```diff
@@ -1,4 +1,4 @@
-# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+# (C) Copyright 2021 Hewlett Packard Enterprise Development LP
```

Comment:
Please revert back copy right year to 2020 as no changes taken place in the file, take care of all the resources listed down.

Reply:
No, copyright should be of current year